src/code/code_800E4FE0.c

```diff
@@ -169,9 +150,9 @@ AudioTask* func_800E5000(void) {
     gAudioCtx.audioRandom = gAudioCtx.audioRandom + gAudioCtx.aiBuffers[index][gAudioCtx.totalTaskCount & 0xFF];
 
     // gWaveSamples[8] interprets compiled assembly code as s16 samples as a way to generate sound with noise.
-    // Start with the address of func_800E4FE0, and offset it by a random number between 0 - 0xFFF0
+    // Start with the address of AudioThread_Update(), and offset it by a random number between 0 - 0xFFF0
```

Comment:
There was a comment somewhere saying that having the brackets helps with doxygen, but I can't seem to find that message thread. I also don't have a preference, but I might leave it based on memory, but can remove it

Reply:
Ah, well, this isn't a doxygen comment anyway, doxygen comments are like `///`, `//!`,  `/**`, `/*!`, (... ?)
my reason for prefering `AudioThread_Update` over `AudioThread_Update()` is refer to the function pointer instead of some return value but it's not a big deal

Reply:
I ended up taking it out then, also not a strong preference on my end